src/components/organisms/AppBar.vue

```diff
@@ -104,26 +120,26 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from "vue"
+import { computed, ref } from "vue"
 import { useRoute, useRouter } from "vue-router"
-import { useDark, useToggle } from "@vueuse/core"
 import { useAuthStore } from "@/store/auth"
 import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/vue"
 import { ArrowCircleDownIcon,ArrowCircleUpIcon, MoonIcon, SunIcon } from "@heroicons/vue/outline"
 import { usePodcastStore } from "@/store/podcasts"
+import { useDark, useToggle } from "@vueuse/core"
 
 const authStore = useAuthStore()
 const route = useRoute()
 const router = useRouter()
 const store = usePodcastStore()
-
+const date = ref(new Date().toISOString().substr(0, 10))
```

Comment:
date needs to default to tomorrow

Reply:
this still needs to be tomorrow

Reply:
I have already fixed it.

Please refer -> [https://i.imgur.com/fPV9DzV.png](url)
                      [https://github.com/DaftDoris/newsdesk/pull/229/commits/16171770e650dff9684c17d4bc8421d0448c5d07](url)